src/Elasticsearch/Framework/ElasticsearchFieldBuilder.php

```diff
@@ -121,6 +121,10 @@ private function getCustomFieldsMapping(string $entity, Context $context): array
             $mapping['properties'][$name] = $esType;
         }
 
+        if (empty($mapping['properties'])) {
```

Comment:
```suggestion
        if ($mapping['properties'] === []) {
```

empty is checking too much falsy things. I don't know, if that could lead to unwanted behaviour at this point